.azure/obtemplates/build-linux.yml

```diff
@@ -16,39 +19,39 @@ jobs:
   - task: PowerShell@2
     displayName: Prepare Build Machine
     ${{ if eq(parameters.tls, 'openssl') }}:
-      target: linux_build_container2
+      target:  ${{ variables.image }}-20.04-x86_64
```

Comment:
I don't think this will work in OneBranch. Please try it out via a PR (don't merge) [internally](https://mscodehub.visualstudio.com/_git/msquic). You can use git to add a remote to your current enlistment to push your branch there to try to make a PR. Please work with @csujedihy to get this fixed ASAP. Our internal pipeline is blocked until we do.